src/harness/testcases/registration_testcase.py

```diff
@@ -181,3 +181,69 @@ def test_10_3_4_2_5_3(self):
     # Check registration response
     self.assertFalse('cbsdId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_REG_26(self):
+    """Category Error in Array request (responseCode 202)
+
+    The response should be SUCCESS for the first CBSD,
+    CATEGORY_ERROR for the second, third, and fourth CBSDs.
+    """
+
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    self.assertEqual(device_1['cbsdCategory'], 'A')
+    self.assertEqual(device_2['cbsdCategory'], 'A')
```

Comment:
Consider putting this in after line 207 where it's clearer that device_2 needs to be cat. A?